ward/terminal.py

```diff
@@ -40,11 +39,28 @@ def multiline_description(s: str, indent: int, width: int) -> str:
     return rv
 
 
-def output_test_result_line(test_result: TestResult):
-    colour = outcome_to_colour(test_result.outcome)
-    bg = f"on_{colour}"
-    padded_outcome = f" {test_result.outcome.name[:4]} "
+def format_test_id(test_result: TestResult) -> (str, str):
+    """
+    Format module name, line number, and test case number
+    """
+
+    test_id = lightblack(
+        f"{format_test_location(test_result)}{format_test_case_number(test_result)}:"
+    )
+
+    return test_id
+
+
+def format_test_location(test_result: TestResult) -> str:
+    """
```

Comment:
There's a couple of empty docstrings

Reply:
Whoops. That's a habit of mine to remind myself to document later and keep stub functions valid.